### PR TITLE
Feature/#12 font awesome icons

### PR DESCRIPTION
**Pull request by JSchraff**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,3 +6,11 @@ If you want to contribute please follow this guideline.
 
 For editing fxml files, use the "JavaFX Scene Builder" in version 8.5.0.  
 Layouting is generally done with fxml files.
+
+For fontawesome support add Libraries 
+`de.jensd:fontawsomefx-commons:8.15`
+`de.jensd:fontawsomefx-fontawsome:4.7.0-5`
+to the "JavaFX Scene Builder"
```
Comment (Death111) [inline CONTRIBUTING.md]: are there instructions on how to add a library which we can reference?

File: src/main/resources/layouts/ViewLayout.fxml [hunk 2]
```diff
@@ -52,22 +51,14 @@
...
                                  </graphic>
                               </Button>
                               <Button fx:id="settingsButton" mnemonicParsing="false" prefHeight="25.0" prefWidth="22.0" stylesheets="@../css/application.css">
                                  <graphic>
-                                    <ImageView fitHeight="24.0" fitWidth="22.0" pickOnBounds="true" preserveRatio="true">
-                                       <image>
-                                          <Image url="@../icons/icon_gear.png" />
-                                       </image>
-                                    </ImageView>
+                                    <FontAwesomeIconView fx:id="settingsIcon" glyphName="COG" glyphStyle="fas" size="20" />
```
Comment (Death111) [inline src/main/resources/layouts/ViewLayout.fxml]: are there less 'bold' icons available?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -6,3 +6,11 @@ If you want to contribute please follow this guideline.
```
Comment (Death111) [inline CONTRIBUTING.md]: unrelated to this file:

- can we use a Seach icon in the Search field?
- Should we use a + icon on the "add new project" button?

**Comment by Death111** [comment]
When #46 is merged, also change Copy to clipboard and Edit button to icons in ReportStage

File: src/main/resources/layouts/ViewLayout.fxml (hunk 2 above)
Comment (JSchraff) [reply src/main/resources/layouts/ViewLayout.fxml]: not for free

File: CONTRIBUTING.md (hunk 3 above)
Comment (JSchraff) [reply CONTRIBUTING.md]: can put search Icon infront of search field

could use + icon , no big deifference

**Pull request merged**